src/main/java/it/pagopa/pn/deliverypush/action/analogworkflow/AnalogWorkflowPaperChannelResponseHandler.java

```diff
@@ -152,7 +159,7 @@ else if (timelineElementInternal.getDetails() instanceof BaseAnalogDetailsInt se
             ResponseStatusInt status = mapPaperStatusInResponseStatus(response.getStatusCode());
 
             final String prepareRequestId = timelineElementInternal.getElementId();
-            String sendRequestId = paperChannelUtils.getSendRequestId(response.getIun(), prepareRequestId);
+            String sendRequestId = paperChannelUtils.getSendRequestId(response.getIun(), prepareRequestId, TimelineElementCategoryInt.SEND_ANALOG_DOMICILE);
             
```

Comment:
togliere parametro aggiuntivo, vedi commento in paperChannelUtils

Reply:
resolved with commit ac0d3f71ccdd88155587af92c33c47243ad239e1